src/main/java/de/dafuqs/spectrum/enchantments/CloversFavorEnchantment.java

```diff
@@ -37,10 +44,23 @@ public int getMaxPower(int level) {
 	public int getMaxLevel() {
 		return SpectrumCommon.CONFIG.CloversFavorMaxLevel;
 	}
+
+	@Override
+	public boolean isAcceptableItem(ItemStack stack) {
+		var item = stack.getItem();
+		if(SpectrumIntegrationPacks.isIntegrationPackActive(SpectrumIntegrationPacks.MALUM_ID) &&
+				item instanceof MalumScytheItem)
+		{
+			return true;
+		}
+
+		return super.isAcceptableItem(stack);
+	}
 	
 	@Override
 	public boolean canAccept(Enchantment other) {
-		return super.canAccept(other) && other != Enchantments.LOOTING;
+		return super.canAccept(other) && other != Enchantments.LOOTING &&
+				!(SpectrumIntegrationPacks.isIntegrationPackActive(SpectrumIntegrationPacks.MALUM_ID) && other == EnchantmentRegistry.SPIRIT_PLUNDER.get());
```

Comment:
Does this run in an instance without Malum installed?
Mentioning `EnchantmentRegistry.SPIRIT_PLUNDER` may throw a class load error.

Reply:
it runs perfectly fine, actually!